tests/Unit/Adapter/Tax/TaxComputerTest.php

```diff
@@ -1,151 +0,0 @@
-<?php
-/**
- * Copyright since 2007 PrestaShop SA and Contributors
- * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
- *
- * NOTICE OF LICENSE
- *
- * This source file is subject to the Open Software License (OSL 3.0)
- * that is bundled with this package in the file LICENSE.md.
- * It is also available through the world-wide-web at this URL:
- * https://opensource.org/licenses/OSL-3.0
- * If you did not receive a copy of the license and are unable to
- * obtain it through the world-wide-web, please send an email
- * to license@prestashop.com so we can send you a copy immediately.
- *
- * DISCLAIMER
- *
- * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
- * versions in the future. If you wish to customize PrestaShop for your
- * needs please refer to https://devdocs.prestashop.com/ for more information.
- *
- * @author    PrestaShop SA and Contributors <contact@prestashop.com>
- * @copyright Since 2007 PrestaShop SA and Contributors
- * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
- */
-
-declare(strict_types=1);
-
-namespace Tests\Unit\Adapter\Tax;
-
-use Generator;
-use PHPUnit\Framework\MockObject\MockObject;
-use PHPUnit\Framework\TestCase;
-use PrestaShop\Decimal\DecimalNumber;
-use PrestaShop\PrestaShop\Adapter\Tax\TaxComputer;
-use PrestaShop\PrestaShop\Adapter\TaxRulesGroup\Repository\TaxRulesGroupRepository;
-use PrestaShop\PrestaShop\Core\Domain\Country\ValueObject\CountryId;
-use PrestaShop\PrestaShop\Core\Domain\TaxRulesGroup\ValueObject\TaxRulesGroupId;
-
-class TaxComputerTest extends TestCase
```

Comment:
Why removed whole test? The class still exists, couldn't it be adjusted?

Reply:
because unit test doesn't have database and the tax calculator has sql inside it we can't test it with unit test. To make it testable it would require lot of refactoring. If we want to do it i would prefer new PR for that.

Reply:
I disagree the class was previously unit testable because we mocked the query results If it's not possible anymore it probably means the architecture has been improperly modified (but I already commented on this part earlier ^^)

Reply:
We mocked it because we used custom repository to get basic tax group without any calculations. Now we use TaxManagerFactory that has to be called static for each tax group and it has sql inside it. So there is no way to mock it. Atleast i don't know how to mock it without being able to inject it into constructor.

Reply:
You're right with the latest changes it's not mockable anymore. Then we should move this test into integration tests, so that we have access to the database and taxes installed with the default features. This will allow us to perform a few tests based on the data from fixtures.